docusaurus.config.js

```diff
@@ -23,14 +23,21 @@ module.exports = {
       },
       items: [
         {
-          to: 'docs/',
```

Comment:
Did you remove the docs links ? I think they should be there.

Reply:
Yes, I removed it following the problem's suggestion (#72), because the "docs" are already in the "get started" button.

Reply:
> 

Sure.

Reply:
The replacement of the text by the social media icons, was a suggestion given by David in the fix comments.

Reply:
![image](https://user-images.githubusercontent.com/6617759/149817232-91580f3c-687f-4665-adf5-4667a01fb648.png)

It seems the buttons are now well aligned, the space between the first and second buttons is not the same in the third and fourth buttons.

Reply:
+ 1

Reply:
The spacing has been corrected. 
What do you think? :) 

![image](https://user-images.githubusercontent.com/93731961/149821911-82964c1a-f850-4597-a5dd-7c48ad17dc5b.png)